src/@next/TextArea/TextAreaStyle.ts

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import { borderRadius4 } from '../utilities/borderRadius';
+import { Neutral, Red } from '../utilities/colors';
+import { NotoSans } from '../utilities/fonts';
+
+export const StyledTextAreaContainer = styled.div`
+  position: relative;
+  width: 100%;
+  max-width: 520px;
+
+  border: 1px solid ${Neutral.B68};
+  border-radius: ${borderRadius4};
+  padding: 8px 12px 25px 12px;
```

Comment:
equivalent of 8px padding-x and 12px padding-y; the 25px is 12px for padding bottom and 17px height to put the character count (which I used absolute positioning as I feel it's easier)